common/user/session.ts

```diff
@@ -15,6 +15,8 @@ export const UNAUTHENTICATED_USER = {
     firstname: '',
     lastname: '',
     userID: '-/-',
+    lastTimeCheckedNotifications: new Date(),
```

Comment:
I think it's wrong - all present notifications would be older than the current timestamp

Reply:
This is an unauthorized user, I just set a value to satisfy TS an unauthorized user shouldn't have any notifications.